spec/requests/volunteers_spec.rb

```diff
@@ -106,8 +106,8 @@
     it "shows correct supervisor options", :aggregate_failures do
       supervisors = create_list(:supervisor, 3, casa_org: organization)
 
-      page = request.parsed_body
-      supervisors.each { |supervisor| expect(page).to include(supervisor.display_name.html_safe) }
+      page = CGI.unescapeHTML(request.parsed_body)
+      supervisors.each { |supervisor| expect(page).to include(supervisor.display_name) }
```

Comment:
It's not realistic to test the contents of the page as though it were not html escaped. Could you change the test to expect the HTML escaped text in the unchanged page? It also takes more effort to convert the whole page into an unescaped form

Reply:
thanks, that makes sense!